pro/src/screens/OfferEducationalStock/__specs__/OfferEducationalStock.spec.tsx

```diff
@@ -0,0 +1,218 @@
+
+const defaultProps: IOfferEducationalStockProps = {
+  initialValues: DEFAULT_EAC_STOCK_FORM_VALUES,
+  offer: collectiveOfferFactory({}),
+  onSubmit: jest.fn(),
+  mode: Mode.CREATION,
+}
+
+const initialValuesNotEmpty = {
+  ...DEFAULT_EAC_STOCK_FORM_VALUES,
+  eventDate: new Date('2023-03-30T00:00:00.000Z'),
+  eventTime: new Date('2023-03-30T00:00:00.000Z'),
+  bookingLimitDatetime: new Date('2023-03-30'),
+  numberOfPlaces: 10,
+  totalPrice: 100,
+}
+
+const mockedUsedNavigate = jest.fn()
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedUsedNavigate,
+}))
+
+describe('OfferEducationalStock', () => {
+  it('should render for offer with a stock', () => {
+    const offer = collectiveOfferFactory()
+    const testProps: IOfferEducationalStockProps = {
+      ...defaultProps,
+      offer,
+      initialValues: {
+        eventDate: new Date('2022-02-10T00:00:00.000Z'),
+        eventTime: new Date('2022-02-10T00:00:00.000Z'),
+        bookingLimitDatetime: new Date('2022-02-10'),
+        numberOfPlaces: 10,
+        totalPrice: 100,
+        priceDetail: 'Détail du prix',
+        educationalOfferType: EducationalOfferType.CLASSIC,
+      },
+      mode: Mode.EDITION,
+    }
+    renderWithProviders(<OfferEducationalStock {...testProps} />)
+
+    screen.getByText('Date et prix')
+  })
+
+  it('should render for offer with a stock', () => {
+    const offer = collectiveOfferFactory({ isPublicApi: true })
+    const testProps: IOfferEducationalStockProps = {
+      ...defaultProps,
+      offer,
+      mode: Mode.EDITION,
+    }
+    renderWithProviders(<OfferEducationalStock {...testProps} />)
+
+    expect(
+      screen.getByText('Offre importée automatiquement')
+    ).toBeInTheDocument()
+  })
+
+  it('should not disable price and place when offer status is reimbursment', () => {
+    const offer = collectiveOfferFactory({ isPublicApi: false })
+    const testProps: IOfferEducationalStockProps = {
+      ...defaultProps,
+      offer,
+      mode: Mode.READ_ONLY,
+    }
+    renderWithProviders(<OfferEducationalStock {...testProps} />)
+
+    const priceInput = screen.getByLabelText('Prix global TTC')
+    const placeInput = screen.getByLabelText('Nombre de places')
+
+    expect(priceInput).not.toBeDisabled()
+    expect(placeInput).not.toBeDisabled()
+  })
+  it('should post form is value are correct', async () => {
+    const offer = collectiveOfferFactory({ isPublicApi: false })
+    const testProps: IOfferEducationalStockProps = {
+      ...defaultProps,
+      offer,
+      initialValues: initialValuesNotEmpty,
+      mode: Mode.CREATION,
+    }
+    renderWithProviders(<OfferEducationalStock {...testProps} />)
+    const submitButton = screen.getByRole('button', { name: 'Étape suivante' })
+    await userEvent.click(submitButton)
+
+    expect(testProps.onSubmit).toHaveBeenCalled()
+  })
+  describe('wrong students modal', () => {
```

Comment:
Chaud pour avoir des espaces entre les `it`/`describe` :smile:

Reply:
🙈 🙈 
Il faudra presque une règle dans le linter pour qu'il le fasse tout seul 👀